packages/order-backend/src/subscription/service.ts

```diff
@@ -82,6 +82,31 @@ export const getSubscription = async (p: {
   }
 }
 
+export const listSubscriptions = async (p: {
+  orderId: string
+  user: string
+}): Promise<SubscriptionsList> => {
+  const { orderId, user: userId } = p
+  checkBackendUrlExists()
+
+  const url = `${process.env.ORDER_BACKEND_URL}/subscription-get-by-order-id`
+  try {
+    const result = await axios.get<SubscriptionsList>(url, {
```

Comment:
Does the backend not return a list of subscriptions instead of this type SubscriptionList, which is an object?

Reply:
SubscriptionList seems to be kinda not needed interface as it only includes specification for array of subscriptions. Could just use `Subscription[]` here.

Reply:
Good points, I will delete this and use Subscription[] instead. Thanks!